  INFO root [2019-06-10 19:38:57,614] - 
  INFO root [2019-06-10 19:38:57,623] - -----------------------------------------------------------------------------------------


  INFO root [2019-06-10 19:38:57,624] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-06-10 19:38:57,625] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/server/;D:/kbengine/server/scripts/;D:/kbengine/server/res/
  INFO root [2019-06-10 19:38:57,625] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-06-10 19:38:57,717] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-06-10 19:38:57,718] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-06-10 19:38:57,718] - invalid UID(0) <= 0, please check UID for environment! automatically set to 3960.
  INFO root [2019-06-10 19:38:57,728] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 3960. PID: 8240 ----
  INFO root [2019-06-10 19:38:57,730] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-06-10 19:38:57,737] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-06-10 19:38:57,737] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-06-10 19:38:57,938] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-06-10 19:38:57,938] - Script::install(): KBExtra is successfully!
S_INFO root [2019-06-10 19:38:57,965] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-06-10 19:38:57,966] - TelnetServer server is running on port 33000
  INFO root [2019-06-10 19:38:58,330] - ---- interfaces is running ----
 DEBUG root [2019-06-10 19:38:58,334] - Components::process(): Request for the process of identity...
  INFO root [2019-06-10 19:39:04,513] - ServerApp::onRegisterNewApp: uid:3960, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:51542, extaddr:nonsupport, extport:0,  from 127.0.0.1:51547/0/0/0.
  INFO root [2019-06-10 19:39:04,513] - Components::addComponent: reset orderLog, uid=3960!
  INFO root [2019-06-10 19:39:04,513] - Components::addComponent[dbmgr], uid=3960, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-06-10 19:39:05,582] - Components::process(): Found all the components!
  WARN root [2019-06-10 19:39:38,281] - TCPPacketReceiver::processPendingEvents(127.0.0.1:51547): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-06-10 19:39:38,282] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
